src/gui/rss/articlelistwidget.cpp

```diff
@@ -55,6 +55,7 @@ void ArticleListWidget::setRSSItem(RSS::Item *rssItem, bool unreadOnly)
 {
     // Clear the list first
     clear();
+    m_rssArticleToListItemMapping.clear();
```

Comment:
>Well, in an ideal word every non-const member of a class calls this->checkInvariant();

Apparently, we should think about creating some sort of "Best Practices" in addition to "Coding Guidelines". Some things can really make life easier.